internal/app/thanos-notif/app.go

```diff
@@ -16,25 +21,192 @@ type Notifier interface {
 }
 
 const (
-	TransferEventABI = "Transfer(address,address,uint256)"
+	ETHDepositInitiatedEventABI      = "ETHDepositInitiated(address,address,uint256,bytes)"
+	ETHWithdrawalFinalizedEventABI   = "ETHWithdrawalFinalized(address,address,uint256,bytes)"
+	ERC20DepositInitiatedEventABI    = "ERC20DepositInitiated(address,address,address,address,uint256,bytes)"
+	ERC20WithdrawalFinalizedEventABI = "ERC20WithdrawalFinalized(address,address,address,address,uint256,bytes)"
+	DepositFinalizedEventABI         = "DepositFinalized(address,address,address,address,uint256,bytes)"
+	WithdrawalInitiatedEventABI      = "WithdrawalInitiated(address,address,address,address,uint256,bytes)"
+)
+
+const (
+	usdcSepolia = "0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238"
```

Comment:
I guess you list usdc addresses here to get decimals for each network later?
If yes, I think It should be better if we config it in the env and query token info like decimals, token name by calling smart contract functions

Reply:
you're right. The reason for listing token addresses was to obtain decimals and symbol! Do you think it would be better to retrieve it through a smart contract call??

Reply:
I think so, it will make the code more flexible. For example, when there are some new token addresses, we just need to config in ENV, we dont need to update the code again. 

And we only need to obtain token info when the app is starting running. We don't need to make call for every time

Reply:
I understand!  Thanks for the advice 🙌  @nguyenzung